### PR TITLE
wownero: bundle Boost 1.85

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/w/wownero.rb [hunk 1]
```diff
@@ -50,7 +49,35 @@ class Wownero < Formula
 
   conflicts_with "monero", because: "both install a wallet2_api.h header"
 
+  # Currently not compatible with boost 1.86.0 even with `monero` fix
+  # https://github.com/monero-project/monero/commit/ed955bf751e304569cd4c04f558360154e19610e
+  resource "boost" do
+    url "https://github.com/boostorg/boost/releases/download/boost-1.85.0/boost-1.85.0-b2-nodocs.tar.xz"
+    sha256 "09f0628bded81d20b0145b30925d7d7492fd99583671586525d5d66d4c28266a"
+  end
```
Comment (carlocab) [inline Formula/w/wownero.rb:57]: Is there an upstream issue for this? Probably doesn't hurt to keep `boost@1.85` for a bit longer.

Comment (cho-m) [reply Formula/w/wownero.rb:57]: No. Though, I wasn't able to create an account upstream (HTTP 500 errors on sign-in page) - https://git.wownero.com/wownero/wownero/issues

Part of fix requires upstream syncing with `monero` while rest requires fixes in their own code.

`wownero` is the only remaining usage of `boost@1.85` so it essentially determines when we deprecate it.

Comment (p-linnane) [reply Formula/w/wownero.rb:57]: `wownero` has had under 700 installs in the last year. If we want to get rid of `boost@1.85` let's just deprecate it.

**Comment by cho-m** [comment]
Closing for now. Can defer deprecation decision until Boost 1.87.0 release or if someone wants to create PR.

**Pull request closed without merging**